bertytech/layouts/partials/footer.html

```diff
@@ -27,20 +27,20 @@ <h5><a href="{{ cond $is_404 (absURL $url) $url }}">Blog</a></h5>
 						<div class="col-4 col-links">
 							<h5>Developers</h5>
 							<ul class="list-unstyled text-small">
-								{{ $url := relLangURL "/docs" }}
-								<li><a href="{{ cond $is_404 (absURL $url) $url }}">Docs</a></li>
+								<!-- {{ $url := relLangURL "/docs" }}
```

Comment:
I delete this link because it wasn't present in the new design

![image](https://github.com/berty/www.berty.tech/assets/33094556/64626ce7-3733-4f2a-b762-91ee9048b89d)